wiki/pygbag/README.md

```diff
@@ -0,0 +1,153 @@
+
+asyncio.run( main() )
+```
+You don't need to understand what this code does, for now, as this is just some test code to check if you have the right setup.
+
+### Running your code with pygbag
+Now open your command line and and get into the parent directory of the folder you had created.
+> Parent directory as in the directory which holds the folder.
+Now type this command into it and press enter<br>
+`pygbag folder_name`<br>
+replace folder_name with the name that you gave to your game folder.
+
+After running this command go to your preferred web browser, and go to https://localhost:8000 and it should show something like
+![image](https://user-images.githubusercontent.com/78538391/169882643-a93622e2-99fe-4f71-90ed-017ab2da51c6.png)
+> Don't worry if you didn't get the same result, just try redoing the previous steps, and if even that doesn't work, you can feel free to join the [pygame discord server](https://discord.gg/653AkjMd) and ask your queries there!
+
+If you were able to complete this step, congratulations you were successfully able to setup pygbag and also able to test it out.
+***
+
+# *Templates*
+## In this chapter you will learn about templates in pygbag
+So, you might be wondering that how will the players play your game if your window looks so small and the console takes up so much of place. To fix this, you can change the layout of the webpage using templates.
+
+### Using other templates
+So currently, pygbag uses the default template which u might wanna change, for that there are 2 ways you can try
+#### 1. Using the in-built templates
+For that u can use the following command<br>
+`pygbag --template window-template.tmpl folder_name`<br>
+this would now use the window-template which is prebuilt in pygbag and this template would clear everything except the window, from the webpage and align your game window to the top right corner.
+
+#### 2. Installing a template and customizing it
+For this u can go to the static folder in the [pygbag repo](https://github.com/pmp-p/pygbag/tree/main/static) and from there install one of the templates and then you can use the following command:<br>
+`pygbag --template template-name.tmpl test`<br>
+replace template-name with the name you gave to the .tmpl file and running tht would port your game to web with your desired template
+<br><br>
+Using the second approach would be better and also recommended, as in this approach, you can customize the template as per your choice and test out how would they look.(ps. the files are basically html files with internal styling and  internal javascript scripting)
+>NOTE: Almost all line are important, so please think a lot before deleting a line which might seem important, cause even one important line could break the whole thing
+***
+
+# *Project Setup*
+## In this chapter you will learn how to setup your project
+
+### Use assets
+So to include and use an asset, you would need to ad all the assets inside an assets folder. That would fix you program.
```

Comment:
> you would need to ad all the assets inside an assets folder. That would fix you program.

change to :
 You need put all your assets  alongside main.py or in a subfolder. Your game must be all contained inside the folder that host "main.py" including the pure python modules ( mention that on next block too ) you want to use and are not in PyGame Web prebuilts ( check import in terminal available here https://pygame-web.github.io/pygbag/python311.html )

make a note on assets to expand that they are : image, music , json files , maybe databases and be explicit that they are 
writeable but not saved when game exit. Also explain what format are preferred for web eg ( jpg / ogg  smaller than bmp/wav )